helpers/setup.py

```diff
@@ -17,13 +18,27 @@ def run(cls, config):
         """
 
         if not os.path.isdir("{}/.git".format(config["kobodocker_path"])):
+            # Move unique id file to /tmp in order to clone without errors
+            # (e.g. not empty directory)
+            uniqid_mv_command = [
+                "mv", Config.UNIQUE_ID_FILE,
+                "/tmp"
+            ]
+            CLI.run_command(uniqid_mv_command, cwd=config["kobodocker_path"])
+
             # clone project
             git_command = [
                 "git", "clone", "https://github.com/kobotoolbox/kobo-docker",
                 config["kobodocker_path"]
             ]
             CLI.run_command(git_command, cwd=os.path.dirname(config["kobodocker_path"]))
 
+            uniqid_restore_command = [
+                "mv", "/tmp/{}".format(Config.UNIQUE_ID_FILE),
```

Comment:
`os.path.join` / can we use https://docs.python.org/2/library/os.html#os.rename instead of platform-dependent `mv`?